src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/FilingsService.java

```diff
@@ -92,113 +93,116 @@ public FilingsService(
   }
 
   public FilingApi generateOverseasEntityFiling(
-      String requestId,
-      String overseasEntityId,
-      Transaction transaction,
-      String passThroughTokenHeader)
-      throws SubmissionNotFoundException, ServiceException {
+          String requestId,
+          String overseasEntityId,
+          Transaction transaction,
+          String passThroughTokenHeader)
+          throws SubmissionNotFoundException, ServiceException {
     var filing = new FilingApi();
     filing.setKind(FILING_KIND_OVERSEAS_ENTITY);
     setFilingApiData(filing, requestId, overseasEntityId, transaction, passThroughTokenHeader);
     return filing;
   }
 
   private void setFilingApiData(
-      FilingApi filing,
-      String requestId,
-      String overseasEntityId,
-      Transaction transaction,
-      String passThroughTokenHeader)
-      throws SubmissionNotFoundException, ServiceException {
+          FilingApi filing,
+          String requestId,
+          String overseasEntityId,
+          Transaction transaction,
+          String passThroughTokenHeader)
+          throws SubmissionNotFoundException, ServiceException {
     var logMap = new HashMap<String, Object>();
     logMap.put(OVERSEAS_ENTITY_ID_KEY, overseasEntityId);
     logMap.put(TRANSACTION_ID_KEY, transaction.getId());
 
     Map<String, Object> userSubmission = new HashMap<>();
 
+//    OverseasEntitySubmissionDto submissionDto = setSubmissionData(userSubmission, overseasEntityId, logMap);
     OverseasEntitySubmissionDto submissionDto =
-        setSubmissionData(userSubmission, overseasEntityId, logMap);
-    setPaymentData(userSubmission, transaction, passThroughTokenHeader, logMap);
+            overseasEntitiesService.getOverseasEntitySubmission(overseasEntityId)
```

Comment:
setSubmissionData() is now for registration flow only.
setUpdateSubmissionData() is for updates.
They both populate different models in the end. Registration converts submissionDto to a Map to put in the **data** Map object of FilingApi, whereas Update just puts submissionDto straight into UpdateSubmission.userSubmission because they are both of type OverseasEntitySubmissionDto, so no conversion needed when its an update.